talkative_task_robot.rb

```diff
@@ -51,7 +13,7 @@
 last_task = "last task"
 to_dos << last_task
 puts(" You just remembered you want to do something when all the other stuff is done #{last_task}")
-user = {}
+user = {name: "Greg"}
```

Comment:
Whenever you use braces you insert a space after the opening and before the closing.

`{ name: "Greg" }`

Also, you should be asking for the user's name rather than hard code it.